steps/measure.py

```diff
@@ -121,6 +121,7 @@ def evaluate_ansatz_based_cost_function(
     noise_model: Optional[str] = None,
     device_connectivity: Optional[str] = None,
     prior_expectation_values: Optional[str] = None,
+    estimation_tasks_transformations_kwargs: Optional[Dict] = {},
```

Comment:
You shouldn't use an empty dict (or list) as a default (see SO ). Long story short – it leads to errors that might be hard to track.
Its better to do:
```
def fun(params=None):
    if params is None:
        params = {}
```

Reply:
Should have answered in that conversation, sorry!

Reply:
Ooh thanks for the tip! Fixed in 31f2a9ade7f1581587315afe197e356598114812